build.gradle.kts

```diff
@@ -50,7 +51,10 @@ subprojects {
         mavenCentral()
         maven {
             url = uri("https://packages.confluent.io/maven/")
-            group = "io.confluent"
+            content {
+                includeGroup("io.confluent")
```

Comment:
This limits what's pulled from the Confluent repo to group `io.confluent` (and `org.apache.kafka`, which is also required to build with `-ce` versions).